v0.2.1 released with native JSON support

dnet opened the pull request:
I merged your metadata commits and changed the version number, so it should be fine.

This version improves usability for requests with a JSON body, since v0.1 represents these as dumb strings, while v0.2.1 converts them to Python literals using native types (for example `null` becomes `None`), and leaves JSON serialization to `requests` which supports it since 2014. This results in editors showing syntax highlighting for the payloads and easier modification in scripts.

dnet committed:
processBody: added intermediate variable contentType

dnet committed:
processBody: separated appending "data="

dnet committed:
processBody: replaced boolean return value with enum

dnet committed:
added support for native JSON in Requests

Apparently, it's been available since 2014:
https://github.com/requests/requests/commit/8f17741849edb5e7eba0356f90cb17e43c938a2f

dnet committed:
Merged PortSwigger's version with Burp metadata

dnet committed:
bumped version number in Burp manifest as well

pajswigger commented:
Unfortunately this introduces a code injection vulnerability. Consider a request like this:

```
{"foo": "bar" }
print("i love burp") #
```

dnet committed:
ignore Content-Length header as well

dnet committed:
replaced simple transformer with full-blown parser

to thwart code injection vulnerabilities, however remote they might be,
see https://github.com/PortSwigger/copy-as-python-requests/pull/1

thx @pajswigger

dnet commented:
@pajswigger I added commit e24981a, I hope you consider this a solution for the issue you raised.

pajswigger commented:
Looks good to me! Your updated BApp is on the Store
